apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/Indictment.tsx

```diff
@@ -105,6 +108,13 @@ const Indictment = () => {
   const [demandsErrorMessage, setDemandsErrorMessage] = useState('')
   const [civilDemandsErrorMessage, setCivilDemandsErrorMessage] = useState('')
 
+  // Use the gender of the single defendant if there is only one,
+  // otherwise default to male
+  const gender =
```

Comment:
This is duplicated logic from earlier, can we have one function which takes in defendants and fetches the gender?

Reply:
I'll consider doing that in my next PR.